src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -160,6 +160,12 @@ internal static class LoggerExtension
             new EventId(324, nameof(AutoRecoveringFileSystemWatcherUnableToRecover)),
             "Unable to recover (path: '{path}')");
 
+        private static readonly Action<ILogger, string, string, Exception> _scriptStartUpLoadedStartUpExtension =
+            LoggerMessage.Define<string, string>(
+            LogLevel.Information,
+            new EventId(325, nameof(ScriptStartUpLoadedStartUpExtension)),
+            "Loaded startup extension '{startupExtensionName}' ({startupExtensionVersion})");
+
```

Comment:
nit: You can probably remove "startup" from the message -- `"Loaded extension '{extensionName}' ({extensionVersion})"`